Makefile

```diff
@@ -3,8 +3,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 VERSION                       := $(shell cat VERSION)
-REGISTRY                      := eu.gcr.io/gardener-project/gardener
-PREFIX                        := vpn
+REGISTRY                      := europe-docker.pkg.dev/gardener-project/public
+PREFIX                        := gardener/vpn
```

Comment:
Please add `gardener` to `REGISTRY` and remove it from `PREFIX`.

Reply:
done

Reply:
Thanks.